otx/algorithms/common/tasks/training_base.py

```diff
@@ -57,7 +57,7 @@
 
 logger = get_logger()
 TRAIN_TYPE_DIR_PATH = {
-    TrainType.INCREMENTAL.name: ".",
+    TrainType.INCREMENTAL.name: "",
```

Comment:
@harimkang Is it okay to change this. AFAIK there is specific reason for this from building side